chatsky/core/ctx_utils.py

```diff
@@ -64,6 +64,11 @@ class FrameworkData(BaseModel, arbitrary_types_allowed=True):
     "Enables complex stats collection across multiple turns."
     slot_manager: SlotManager = Field(default_factory=SlotManager)
     "Stores extracted slots."
+    response_exception: Optional[str] = Field(default=None, exclude=True)
+    """
+    Stores exception messages raised from response functions wrapped in
+    :py:class:`~chatsky.processing.standard.AddFallbackResponses`.
+    """
```

Comment:
I think that this is too random of a field that is unusable in many cases.

Better approach seems to be to make this field hold last exception raised from any script function and not only responses when `AddFallbackResponses` is used.

```python
class BaseScriptFunction:
  async def wrapped_call(ctx):
    ...
    except Exception as exc:
      ctx.framework_data.last_exception = exc
```

Preferably in a separate PR.